### PR TITLE
Async support for HazelcastSessionManager

### PR DESCRIPTION
**Pull request by emre-aydin**

### CODE REVIEW

File: tomcat7/src/test/java/com/hazelcast/session/AsyncServlet.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    private static void newEvent() {
+        ArrayList<AsyncContext> clients = new ArrayList<AsyncContext>(queue.size());
+        queue.drainTo(clients);
+        for (AsyncContext ac : clients) {
+            try {
+                ac.getResponse().getWriter().write("OK");
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (pveentjer) [inline tomcat7/src/test/java/com/hazelcast/session/AsyncServlet.java]: Do we want to print a stacktrace or log it? e.printStackTrace should not happen in production code.

File: tomcat8/src/test/java/com/hazelcast/session/AsyncServlet.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    private static void newEvent() {
+        ArrayList<AsyncContext> clients = new ArrayList<AsyncContext>(queue.size());
+        queue.drainTo(clients);
+        for (AsyncContext ac : clients) {
+            try {
+                ac.getResponse().getWriter().write("OK");
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (pveentjer) [inline tomcat8/src/test/java/com/hazelcast/session/AsyncServlet.java]: print stacktrace

File: tomcat85/src/test/java/com/hazelcast/session/AsyncServlet.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    private static void newEvent() {
+        ArrayList<AsyncContext> clients = new ArrayList<AsyncContext>(queue.size());
+        queue.drainTo(clients);
+        for (AsyncContext ac : clients) {
+            try {
+                ac.getResponse().getWriter().write("OK");
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (pveentjer) [inline tomcat85/src/test/java/com/hazelcast/session/AsyncServlet.java]: print stacktrace

File: tomcat7/src/test/java/com/hazelcast/session/AsyncServlet.java (hunk 1 above)
Comment (emre-aydin) [inline tomcat7/src/test/java/com/hazelcast/session/AsyncServlet.java]: This is not production code, it's test code. Same with the other two. `e.printStackTrace` should be fine for test classes.

**Pull request merged**